internal/template-validator/validation/util_test.go

```diff
@@ -0,0 +1,101 @@
+package validation
```

Comment:
Can you restore the original file name: `utils_test.go`? Just to be consistent with other files in the repo, were we use `utils`.